projects/notebooks/roles/notebooks_dashboard_scale_test/tasks/main.yml

```diff
@@ -27,7 +26,14 @@
     test_artifacts_collected: "{{ notebooks_dashboard_scale_test_artifacts_collected }}"
     capture_prom_db: "{{ notebooks_dashboard_scale_test_capture_prom_db }}"
 
+- name: Set system-under-test == driver-cluster if no system-under-test (SUT) is provided
+  set_fact:
+    sut_cluster_kubeconfig: "{{ notebooks_dashboard_scale_test_sut_cluster_kubeconfig | default(lookup('env', 'KUBECONFIG'), true) }}"
+
+
 - name: Fetch RHODS endpoints from the SUT cluster
+  environment:
+    KUBECONFIG: '{{ sut_cluster_kubeconfig }}'
```

Comment:
you don't need this kubeconfig machinery, I think I removed it all when I prepared the ansible role for you